frontend/web/components/metadata/MetadataLinkedTable.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC } from 'react'
+
+type MetadataLinkedTableType = {}
+
+const MetadataLinkedTable: FC<MetadataLinkedTableType> = ({}) => {
+  return (
+    <div>
+      <p>test</p>
```

Comment:
Im guessing this is unused or something isn't pushed

Reply:
Yes, Deleted